app/queries/payments_requiring_action.rb

```diff
@@ -5,10 +5,12 @@ def initialize(user)
     @user = user
   end
 
-  attr_reader :user
-
   def query
     Spree::Payment.joins(order: [:user]).where.not(cvv_response_message: nil).
       where("spree_users.id = ?", user.id)
   end
+
+  private
+
+  attr_reader :user
```

Comment:
Yeah good point. I was modeling it off of the other queries; I suppose if it ever gets more complex having it structured like this could be a good guide.